src/store/global/global.thunks.js

```diff
@@ -86,6 +87,10 @@ function changeRedirectRoute (redirectRoute) {
  */
 function fetchFiatExchangeRates (symbols) {
```

Comment:
Maybe you don't need the parameter anymore?

Reply:
yeah, nice catch!